src/cgroups/v2/memory.rs

```diff
@@ -1,10 +1,16 @@
-use anyhow::Result;
+use anyhow::{Result, *};
```

Comment:
Why the *? Shouldn't 
```suggestion
use anyhow::{bail, Result};
```
be enough?

Reply:
This is probably left behind from something I intended to clean up.